.github/dependabot.yml

```diff
@@ -1,5 +1,10 @@
 version: 2
 updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
```

Comment:
This is a great addition.

Reply:
Do we also want to set `open-pull-requests-limit`?

Reply:
> Do we also want to set `open-pull-requests-limit`?

We can certainly add it, but it hasn't been a problem to date. Since we have grouped PRs, we generally get one or two a week. We almost always get one for general library updates and rarely we get one for docker. I wouldn't anticipate the action PRs piling up either - the version bumps there are pretty infrequent

Reply:
All good then, in the unlikely event it poses an issue we can add it.